lib/utils/fs/readFileSync.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const YAML = require('js-yaml');
+const fse = require('./fse');
+
+function readFileSync(filePath) {
```

Comment:
You could use `this.serverless.utils.readFileSync` and remove this helper function.